src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -0,0 +1,671 @@
+	private JScrollPane scrollPaneInventory;
+	private JTable jTableInventory;
+	private String[] columsNames = {
+			MessageBundle.getMessage("angal.common.reference.label").toUpperCase(),
+			MessageBundle.getMessage("angal.common.date.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.common.status.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.common.user.col").toUpperCase()
+	};
+	private int[] columwidth = { 150, 150, 150, 200 };
+	private Class[] columnClasses = { String.class, String.class, String.class, String.class };
+	private boolean[] columnCentered = { false, true, true, true };
+	private JComboBox<String> statusComboBox;
+	private JLabel statusLabel;
+	private JButton next;
+	private JButton previous;
+	private JComboBox<Integer> pagesComboBox = new JComboBox<>();
+	private JLabel ofPagesLabel = new JLabel(MessageBundle.formatMessage("angal.common.pages.fmt.txt", 1));
+	private static int PAGE_SIZE = 24;
+	private int startIndex = 0;
+	private int totalRows;
+	private MedicalInventoryManager medicalInventoryManager = Context.getApplicationContext().getBean(MedicalInventoryManager.class);
+	private List<MedicalInventory> inventoryList;
+
+	public InventoryBrowser() {
+		initComponents();
+	}
+
+	private void initComponents() {
+		setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
+		setMinimumSize(new Dimension(850, 550));
+		setLocationRelativeTo(null); // center
+		setTitle(MessageBundle.getMessage("angal.inventory.inventorybrowser.title"));
+
+		panelHeader = getPanelHeader();
+		getContentPane().add(panelHeader, BorderLayout.NORTH);
+
+		panelContent = getPanelContent();
+		getContentPane().add(panelContent, BorderLayout.CENTER);
+
+		panelFooter = getPanelFooter();
+		getContentPane().add(panelFooter, BorderLayout.SOUTH);
+
+		addWindowListener(new WindowAdapter() {
+
+			@Override
+			public void windowClosing(WindowEvent e) {
+				dispose();
+			}
+		});
+		pagesComboBox.setEditable(true);
+		previous.setEnabled(false);
+		next.setEnabled(false);
+		next.addActionListener(actionEvent -> {
+			if (!previous.isEnabled()) {
+				previous.setEnabled(true);
+			}
+			startIndex += PAGE_SIZE;
+			int page = startIndex / PAGE_SIZE + 1;
+			jTableInventory.setModel(new InventoryBrowsingModel(page, PAGE_SIZE));
+			if ((startIndex + PAGE_SIZE) > totalRows) {
+				next.setEnabled(false);
+			}
+			pagesComboBox.setSelectedItem(page);
+		});
+		previous.addActionListener(actionEvent -> {
+			if (!next.isEnabled()) {
+				next.setEnabled(true);
+			}
+			startIndex -= PAGE_SIZE;
+			int page = startIndex / PAGE_SIZE + 1;
+			jTableInventory.setModel(new InventoryBrowsingModel(page, PAGE_SIZE));
+			if (startIndex < PAGE_SIZE) {
+				previous.setEnabled(false);
+			}
+			pagesComboBox.setSelectedItem(page);
+		});
+		pagesComboBox.addItemListener(itemEvent -> {
+			int eventID = itemEvent.getStateChange();
+
+			if (eventID == ItemEvent.SELECTED) {
+				int page_number = (Integer) pagesComboBox.getSelectedItem();
```

Comment:
```suggestion
				int pageNumber = (Integer) pagesComboBox.getSelectedItem();
```
local variables should use camel case and not underscores

Reply:
You're right, thanks!